### ISSUE TITLE
Tests for twisted.python should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7748]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7748|
|Type|enhancement|
|Created|2015-01-11 12:36:29Z|
|Branch|https://github.com/twisted/twisted/tree/twisted-python-testcase-names-7748|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [misc_7748.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/d84/d84b2b89c8578c0b17b31157f6637bd4d7153294/98df44624e3a7883625b79bf35c783fc738c0f1c.patch) (13003 bytes) - added by eeshangarg on 2015-01-11 12:54:41Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7748 7748
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_twisted_python_testcase_names_7748 branches/twisted-python-testcase-names-7748
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1420979789915406 1420979789915406
changetime__1422308401576468 1422308401576468
version__None None
owner__eeshangarg eeshangarg
cc__eeshangarg
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_1"></a>|@eeshangarg removed owner|
|-|-|

Hi!

I just added a patch(misc_7748.patch​) that changes all of the tests in twisted.python to use the new style of naming test classes as discussed in ticket [ticket:7032].

Happy New Year! :-)

Thanks & regards,
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

(In [43757]) Branching to twisted-python-testcase-names-7748.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @adiroiban|
|-|-|

Looks good. Thanks!


I will wait for test results and merge if all ok!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)|@adiroiban set owner to @eeshangarg|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set status to `closed`|
|-|-|

(In [43772]) Merge twisted-python-testcase-names-7748: Rename test cases from twisted.python.

Author: eeshangarg
Reviewers: adiroiban
Fixes: [#7748](https://github.com/twisted/twisted/issues/7748)
